docs/pages/kubernetes-access/helm/guides/gcp.mdx

```diff
@@ -0,0 +1,422 @@
+  fully-qualified domain name that you'll use for external access to your Teleport cluster.
+</Admonition>
+
+Once the chart is installed, you can use `kubectl` commands to view the deployment:
+
+```bash
+$ kubectl --namespace teleport get all
+NAME                           READY   STATUS    RESTARTS   AGE
+pod/teleport-b64dd8849-fklvk   1/1     Running   0          7m4s
+pod/teleport-b64dd8849-jqvns   1/1     Running   0          7m15s
+
+NAME               TYPE           CLUSTER-IP     EXTERNAL-IP    PORT(S)                                                      AGE
+service/teleport   LoadBalancer   10.40.14.191   35.203.56.38   443:31758/TCP,3023:30409/TCP,3026:30939/TCP,3024:31403/TCP   7m16s
+
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/teleport   2/2     2            2           7m16s
+
+NAME                                  DESIRED   CURRENT   READY   AGE
+replicaset.apps/teleport-b64dd8849    2         2         2       7m16s
+```
+
+### Set up DNS
+
+You'll need to set up two DNS `A` records: `teleport.example.com` for the web UI, and `*.teleport.example.com`
+for web apps using [application access](../../../application-access/introduction.mdx).
+
+Here's how to do this using Google Cloud DNS:
+
+```bash
+# Change these parameters if you altered them above
+$ NAMESPACE=teleport
+$ RELEASE_NAME=teleport
+
+$ MYIP=$(kubectl --namespace ${NAMESPACE?} get service/${RELEASE_NAME?} -o jsonpath='{.status.loadBalancer.ingress[*].ip}')
+$ MYZONE="myzone"
+$ MYDNS="teleport.example.com"
+
+$ gcloud dns record-sets transaction start --zone="${MYZONE?}"
+$ gcloud dns record-sets transaction add ${MYIP?} --name="${MYDNS?}" --ttl="300" --type="A" --zone="${MYZONE?}"
+$ gcloud dns record-sets transaction add ${MYIP?} --name="*.${MYDNS?}" --ttl="300" --type="A" --zone="${MYZONE?}"
+$ gcloud dns record-sets transaction describe --zone="${MYZONE?}"
+$ gcloud dns record-sets transaction execute --zone="${MYZONE?}"
+```
+
+### Create a Teleport user
+
+At this point, you'll still need to create a user to be able to log into Teleport. This needs to be done on the Teleport auth server,
+so we can run the command using `kubectl`:
+
+```bash
+$ kubectl --namespace teleport exec deploy/teleport -- tctl users add test --roles=access,editor
+User "test" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+https://teleport.example.com:443/web/invite/91cfbd08bc89122275006e48b516cc68
+
+NOTE: Make sure teleport.example.com:443 points at a Teleport proxy which users can access.
+```
+
+Load the user creation link to create a password and set up 2-factor authentication for the Teleport user via the web UI.
+
+You can follow our [Getting Started with Teleport guide](../../../getting-started.mdx#step-2-create-a-teleport-user-and-set-up-two-factor-authentication) to finish setting up your
+Teleport cluster.
+
+#### High availability
```

Comment:
This section is identical to `aws` one? You can move it to `ha.mdx` partial and include using (\! include.mdx \!) directive:

https://goteleport.com/docs/docs/best-practices/#includes